src/datatable.js

```diff
@@ -54,32 +54,34 @@ var defaults = {
     pageSizeSelector: false
 };
 
-var logger = loggerFactory('ui/datatable');
+const logger = loggerFactory('ui/datatable');
 
 /**
  * The CSS class used to hide an element
  * @type {String}
  */
-var hiddenCls = 'hidden';
+const hiddenCls = 'hidden';
 
 /**
  * Deactivate pagination's
+ * @param {Array} pagination
  */
-var disablePaginations = function disablePaginations(paginations) {
-    if (paginations && paginations.length) {
-        _.forEach(paginations, function(pagination) {
-            pagination.disable();
+const disablePagination = pagination => {
+    if (pagination && pagination.length) {
+        _.forEach(pagination, step => {
```

Comment:
from Bertran post: `If the code use lodash where it's not necessary, please use the JavaScript functions (_.forEach(myArray) to myArray.forEach), again only when it's safe to do.`